Dockerfile

```diff
@@ -2,7 +2,13 @@ FROM ruby:2.5.5
 MAINTAINER BioSistemika <info@biosistemika.com>
 
 # additional dependecies
+
+# Add Debian stretch backports repository
+RUN echo 'deb http://http.debian.net/debian stretch-backports main' \
```

Comment:
Maybe it is better to use ´echo "deb http://http.debian.net/debian $(lsb_release -sc)-backports main"´ for more flexibility, in case base image changes